x-pack/filebeat/input/o365audit/auth/cert.go

```diff
@@ -23,26 +24,16 @@ func NewProviderFromCertificate(
 	if err != nil {
 		return nil, fmt.Errorf("failed loading certificates: %w", err)
 	}
-	oauth, err := adal.NewOAuthConfig(endpoint, tenantID)
```

Comment:
The `endpoint` param is no longer used, remove it or underscore? It's only used in one place, and that is internal to the input (though not the package), so suggest remove.

Reply:
Agreed. That's done.